code/datums/greyscale/README.md

```diff
@@ -0,0 +1,91 @@
+# Greyscale Autogenerated Sprites (GAS)
```

Comment:
hes calling it gas the absolute MADMAN

Reply:
Would you prefer Greyscale AutoGenerated Sprites (GAGS)?

Reply:
that sounds funny

Reply:
Both are good